components/supervisor/cmd/rings.go

```diff
@@ -26,6 +26,55 @@ import (
 	"golang.org/x/sys/unix"
 )
 
+var (
+	forwardedSignals = []os.Signal{
```

Comment:
wonder why only these signals

Reply:
that's already really exhaustive. I don't really expect much more than SIGSTOP, SIGCONT and SIGTERM.
Frankly, I've just copied this list from rootlesskit to replicate what we had prior to this change.